lib/active_merchant/billing/gateways/visanet_peru.rb

```diff
@@ -77,7 +77,7 @@ def scrub(transcript)
         transcript.
           gsub(%r((Authorization: Basic )\w+), '\1[FILTERED]').
           gsub(%r((\"cardNumber\\\":\\\")\d+), '\1[FILTERED]').
-          gsub(%r((\"cvv2Code\\\":)\d+), '\1[FILTERED]')
+          gsub(%r((\"cvv2Code\\\":\\\")\d+), '\1[FILTERED]')
```

Comment:
I would have expected a new test here to confirm this functionality? Or a modification to an existing test if it wasn't properly testing scrubbing before?